src/main/java/org/jenkinsci/plugins/sma/SMAJenkinsCIOrgSettings.java

```diff
@@ -0,0 +1,75 @@
+package org.jenkinsci.plugins.sma;
+
+import com.sforce.soap.partner.fault.InvalidSObjectFault;
+import com.sforce.soap.partner.sobject.SObject;
+
+import java.util.Calendar;
+import java.util.NoSuchElementException;
+import java.util.TimeZone;
+
+/**
+ * Created by ronvelzeboer on 08/02/17.
+ */
+public class SMAJenkinsCIOrgSettings {
```

Comment:
@aesanch2 Thanks. Happy to be the first PR and glad to help with extending this plugin and in the end contribute to hopefully increasing the user base of this plugin!

About the JenkinsCISettings that is indeed stored in Salesforce. This is optional and can be enabled by the user. Setting the Global Environment variables to store the last git commit id can cause some issues. Storing the information of the Salesforce org itself has all kind of benefits and is more stable I think. For example if you have your setup on bare metal and want to move to the cloud, you will loose the environment variables. Now you can even run Jenkins from your local machine (next to your normal server) for example and it will still work. In the end, for a successful deploy, all that really matters is the version that is actually on the org. Also I have an extra feature in mind that will use the information in JenkinsCISettings to deal with changes made in the org directly.